.github/workflows/update.yaml

```diff
@@ -0,0 +1,46 @@
+name: Update
+
+# FIXME: Delete when the PR is ready
+on:
+  push:
+    paths: 
+      - "update.yaml"
+
+# Enable for daily auto-updates
+# on:
+#     schedule:
+#     - cron: 07 8 * * * # 08:07 UTC is 00:07 PST
+
+env:
+  ZONEDBOT_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+      
+      - name: Build
+        run: go build -v
+
+      - name: Update data
+        run: make update
```

Comment:
You could add `-mod=readonly` here, but I’m sure there are more commands hidden in `make update` that might still modify the `go.mod` file.

Reply:
Doing a dirty check is probably sufficient.

Reply:
Ok I added this.